tests/test_flow/test_protocols.py

```diff
@@ -372,6 +372,26 @@ def np_array_high_d():
     assert(np.array_equal(arr, desired))
 
 
+def test_yaml_protocol_list(builder):
+    x = ['red', 'blue']
+    @builder
+    @bn.protocol.yaml(default_flow_style=False)
```

Comment:
If `default_flow_style` is an important part of the test, should we validate that it's actually taking effect?  For example, we could export `alist` and check that the YAML looks the way we expect.  This might make sense as a separate  test.